guideforms.py

```diff
@@ -280,7 +280,9 @@
     'experiment_timeline': forms.CharField(
         label='Experiment Timeline', required=False,
         widget=forms.Textarea(attrs={'rows': 2, 'cols': 50, 'maxlength': 1480}),
-        help_text='When does the experiment start and expire?'),
+        help_text=(
+            'For developer trials and origin trials state which versions of '
```

Comment:
I think this should still be for origin trials only?

Reply:
Where are developer trials recorded?

Reply:
They're not.  Here, basically.  Chromestatus will tell you (from status) what features are in developer trial or later. And it's a rough call - there may be functionality checked in prior to this (behind a flag), it's just not "complete".

Reply:
Fair enough.